src/iop/borders.c

```diff
@@ -937,20 +937,23 @@ void gui_init(struct dt_iop_module_t *self)
   gtk_widget_set_tooltip_text(g->size, _("size of the border in percent of the full image"));
 
   DT_BAUHAUS_COMBOBOX_NEW_FULL(g->aspect, self, NULL, N_("aspect"),
-                               _("select the aspect ratio or right click and type your own (w:h)"),
+                               _("select the aspect ratio (right click on slider below to type your own w:h)"),
```

Comment:
Remove `dt_bauhaus_combobox_set_editable(g->aspect, 1);` below?

Reply:
Right, missed that change from the original PR.